client/src/components/PropertiesList.tsx

```diff
@@ -73,7 +72,7 @@ const PropertiesListWithoutI18n: React.FC<
   const addrs = props.state.context.portfolioData.assocAddrs;
   const rsunitslatestyear = props.state.context.portfolioData.searchAddr.rsunitslatestyear;
 
-  const lastColumnRef = createRef<any>();
+  const lastColumnRef = useRef<HTMLDivElement>(null);
```

Comment:
This is the one thing I found that's potentially a bug.  `createRef` is really for class-based components, and will return a brand-new reference every time it's called, whereas `useRef` will return the same one every render.